crutest/src/protocol.rs

```diff
@@ -34,9 +34,11 @@ pub enum CliMessage {
     Export,
     // Run the fill test.
     Fill(bool),
+    // Run the sparse fill test.
+    FillSparse,
     Flush,
     Generic(usize, bool),
-    Info(u64, u64, u64),
+    Info(VolumeInfo, u64, usize),
```

Comment:
We should just pass `VolumeInfo`, because total size and total blocks can be calculated from it.

Reply:
Done.